packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -41,8 +41,11 @@ const POPOVER_PROPS = {
 
 function CopyMenuItem( { blocks, onCopy, label } ) {
 	const ref = useCopyToClipboard( () => serialize( blocks ), onCopy );
-	const copyMenuItemBlocksLabel =
-		blocks.length > 1 ? __( 'Copy blocks' ) : __( 'Copy block' );
+	const copyMenuItemBlocksLabel = _n(
```

Comment:
Looking at the blame history for this change it looks like it was intentionally refactored to not use `_n` https://github.com/WordPress/gutenberg/pull/41886

Reply:
Thanks! Fixed 👍